### DISCUSSION TITLE
Jetson AGX Xavier support

### DISCUSSION THREAD

**Discussion by molohov** [roflcoopter/viseron#488] [Q&A]
Any reason that the Jetson Nano container wouldn't work on another Jetson (specifically the AGX Xavier) with the same SW stack? Has anyone tried this before.

**Comment by roflcoopter** [comment]
Im not sure about this but it might be possible.
The container is pulling libraries from Nvidias Nano repository but i cant see why they would differ much.

If you have an AGX Xavier available i would appreciate if you could try it out!
